src/style.css

```diff
@@ -0,0 +1,169 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&family=Playfair+Display&family=Poppins:wght@200;300;400;500;700&display=swap');
+
+* {
+  margin: 0%;
+  padding: 0%;
+  border: none;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+header,
+footer {
+  height: 30px;
+  background-color: whitesmoke;
+  box-shadow: 2px 2px 5px rgb(218, 116, 133), -2px -2px 2px rgb(228, 89, 112);
+}
+
+p {
+  font-size: 25px;
+  color: rgba(151, 44, 62, 0.877);
+  font-weight: 500;
+  text-shadow: 2px 2px whitesmoke;
+}
+
+h1 {
+  text-align: center;
+  margin-top: 20px;
+  background-image: linear-gradient(-225deg, #ec5242 0%, #44107a 29%, #ec5242 67%, #ec5242 100%);
+  background-clip: border-box;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+td {
+  background: white;
+  border-radius: 20px;
+  padding: 5px 2px;
+  margin: 2px 40px;
+}
+
+button {
+  font-size: 18px;
+  padding: 4px 10px;
+  border-radius: 5px;
+}
+
+button:hover {
+  background: linear-gradient(-225deg, rgb(218, 116, 133) 0%, #44107a 29%, rgb(218, 116, 133) 67%, rgb(231, 9, 46) 100%);
+  color: white;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  height: 90.5vh;
+}
+
+span.user-icon {
+  width: 30px !important;
+  height: 30px;
+  background: url(../image/icon.png) no-repeat;
+  background-size: cover;
+  margin-right: 4px;
+  border-radius: 50%;
+}
+
+.main-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  height: 89.5vh;
+  margin-top: 5px;
+  background: url(../image/9.gif) no-repeat;
+  background-size: cover;
+  box-shadow: inset 0 0 0 2000px rgba(184, 101, 142, 0.2);
+}
+
+.result {
+  /* background: url(../image/9.gif) no-repeat;
+  background-size: 700px 600px; */
+  width: 50%;
+}
+
+.add-data {
+  /* background: url(../image/3.webp) no-repeat;
+  background-size: 700px 620px; */
+  width: 50%;
+}
+
+.recent-score {
+  display: flex;
+  flex-direction: column;
+  width: 65%;
+  margin-top: 60px;
+  height: 70.5vh;
+  margin-left: 150px;
+  box-shadow: 2px 2px 5px rgb(228, 89, 112), -2px -2px 2px rgb(228, 89, 112);
+}
+
+.title {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 8vh;
+  width: 100%;
+  border-bottom: 1px solid rgba(243, 192, 201, 0.925);
+  padding: 4.5px 30px;
+}
+
+.title button {
+  height: 30px;
+}
+
+.recent-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.user {
+  /* background: pink; */
+  text-transform: capitalize;
+  padding-left: 10px;
+  display: flex;
+  flex-direction: row;
+  text-align: center;
+}
+
+.score {
+  /* background: rgb(180, 41, 64); */
+
+  /* color: white; */
```

Comment:
- [OPTIONAL] Please kindly delete the comment line of code to have a good and clean codebase 🙃 🙃